src/module.h

```diff
@@ -0,0 +1,51 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#ifndef MODULE_H
+#define MODULE_H
+
+#include "notificationhandler.h"
+
+#include <QObject>
+
+class QTextStream;
+
+class Module : public QObject {
```

Comment:
Please use Doxygen format for docs. Also, it would be helper to have a docstring for the class itself.